ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -761,7 +758,6 @@ tests:
         and not @fips and @destructive'
       MCE_VERSION: "2.3"
       TAG_VERSION: '@4.13'
-      TEST_ADDITIONAL: Hypershift|Network_Observability
```

Comment:
@LiangquanLi930 should keep Network_Observability?

Reply:
Sorry, Thanks for your reminder